pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/GenericBrokerHostUsageImpl.java

```diff
@@ -85,10 +92,12 @@ private double getTotalCpuLimit() {
     }
 
     private double getTotalCpuUsage() {
-        double cpuUsage = cpuUsageSum / cpuUsageCount;
-        cpuUsageSum = 0d;
-        cpuUsageCount = 0;
-        return cpuUsage;
+        synchronized (this) {
```

Comment:
nit: what about moving the `synchronized` keyword in the method declaration ?

Reply:
I did this, and I also moved the check for `cpuUsageCount` being zero into this method. It makes more sense to me in the synchronized method, and it will ensure that if any other methods end up calling this one, there isn't a chance of accidentally dividing by 0.